src/plugins/data/server/search/strategies/ese_search/ese_search_strategy.test.ts

```diff
@@ -260,6 +260,38 @@ describe('ES search strategy', () => {
 
         expect(mockApiCaller).toBeCalledTimes(0);
       });
+
+      it('should delete when aborted', async () => {
```

Comment:
Do these tests apply to the other search strategies modified in this PR? If so, can we copy the tests to their test files as well?

Reply:
Good catch, I've updated the other strategies as well. You'll notice the logic is slightly different (doesn't check `options.isStored` because the other strategies don't support search sessions).